tests/test_consume_whitespace.py

```diff
@@ -0,0 +1,39 @@
+"""All Unit tests which focus on how whitespace is handled by the calculator.
```

Comment:
I have avoided following Google's recommendation in that case because it makes indentation of docstrings look weird IMHO.

![docstring-indentation](https://cloud.githubusercontent.com/assets/3084059/19839038/6b560858-9e98-11e6-9e31-f0cee1776a4b.png)